src/Wizard/ShopWizard/Services/DefaultSettingsService.php

```diff
@@ -122,19 +122,19 @@ public function hasShippingCountries(): bool
         $shippingCountries = $this->countryRepository->getActiveCountriesList();
         return count($shippingCountries) ? true : false;
     }
-    
+
     /**
      * @return array
      */
-    public function getShippingProfiles()
+    public function getShippingProfiles(): array
```

Comment:
We do not set strict type annotations because it causes problems during the plugin build process in the past.

```suggestion
    public function getShippingProfiles()
```

Reply:
Hi @felixdausch 

I'll be glad to remove those. However at least ShopWizard::structure() would cause an error since it would lose compatibility to its parent in the plugin interface which does have a return type declaration, which results in an error:
https://github.com/plentymarkets/plugin-interface/blob/beta7/Modules/Wizard/Services/WizardProvider.php

Should we remove all the others and leave that one?

Kind regards!